WooCommerce/Classes/ViewRelated/Orders/Order Creation/Taxes/NewTaxRateSelectorViewModel.swift

```diff
@@ -26,16 +26,20 @@ final class NewTaxRateSelectorViewModel: ObservableObject {
     /// Trigger to perform any one time setups.
     let onLoadTriggerOnce: PassthroughSubject<Void, Never> = PassthroughSubject()
 
+    let onTaxRateSelected: (Yosemite.TaxRate) -> Void
+
     /// View models for placeholder rows. Strings are visible to the user as it is shimmering (loading)
     let placeholderRowViewModels: [TaxRateViewModel] = [Int64](0..<3).map { index in
         TaxRateViewModel(id: index, title: "placeholder", rate: "10%")
     }
 
     init(siteID: Int64,
+         onTaxRateSelected: @escaping (Yosemite.TaxRate) -> Void,
```

Comment:
Should we define something like a `typealias TaxRate = Yosemite.TaxRate` in this file so we don't leak implementation details to whoever uses this model?

Reply:
I don't think we're leaking implementation details by exposing the type's library; if the client is going to use it it should be aware that it is coming from `Yosemite`. In this case, the view passes it directly to the EditableOrderViewModel without having to import `Yosemite` there.